docs/utilities/data_classes.md

```diff
@@ -175,7 +175,7 @@ Use **`APIGatewayAuthorizerRequestEvent`** for type `REQUEST` and **`APIGatewayA
 
     @event_source(data_class=APIGatewayAuthorizerRequestEvent)
     def handler(event: APIGatewayAuthorizerRequestEvent, context):
-        user = get_user_by_token(event.get_header_value("Authorization"))
+        user = get_user_by_token(event.headers["Authorization"])
```

Comment:
How about using `event.headers.get()`? So we prevent the customer from having problems with `KeyError`.